hostinfo/hostinfo.go

```diff
@@ -364,6 +368,13 @@ func inDockerDesktop() bool {
 	return false
 }
 
+func inHomeAssistantAddOn() bool {
+	if os.Getenv("SUPERVISOR_TOKEN ") != "" || os.Getenv("HASSIO_TOKEN") != "" {
```

Comment:
There is a space in the environment string `"SUPERVISOR_TOKEN "`, is that intentional?